docs/using-hive.md

```diff
@@ -157,6 +157,102 @@ Credentials will be read from `~/.gcp/osServiceAccount.json`, this can be create
 bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=gcp mycluster
 ```
 
+#### Create Cluster on Bare Metal
+
+Hive supports bare metal provisioning as provided by [openshift-install](https://github.com/openshift/installer/blob/master/docs/user/metal/install_ipi.md)
+
+At present this feature requires a separate pre-existing libvirt provisioning host to run the bootstrap node. This host will require very specific network configuration that far exceeds the scope of Hive documentation. See *LINK TBD* for more information.
+
+![Bare metal provisioning with one hypervisor host per cluster](hive-baremetal-hypervisor-per-cluster.png)
+
+![Bare metal provisioning with a shared hypervisor host](hive-baremetal-shared-hypervisor.png)
+
+To provision bare metal clusters with Hive:
+
+Create a Secret containing a [bare metal enabled InstallConfig](https://github.com/openshift/installer/blob/master/docs/user/metal/install_ipi.md#install-config).
+
+Create a Secret containing the SSH private key that can connect to your libvirt provisioning host, without a passphrase.
+
+```yaml
+- apiVersion: v1
```

Comment:
The `-` at the start of this line and the indentation look suspicious.

Reply:
Yeah probably copied out of a list, bringing it all in one level, thanks!